qemu/tests/qemu_guest_agent.py

```diff
@@ -102,7 +103,42 @@ def _cleanup_open_session(self):
         except Exception:
             pass
 
-    @error.context_aware
+    @error_context.context_aware
+    def setup_check_ga_install(self, params, vm):
+        '''
+        Check if the package is installed
+        :param params: Dictionary with the test parameters
+        :param vm: the vitual machine for test
+        '''
+        error_context.context("Check whether qemu-ga is installed.", logging.info)
+        session = self._get_session(params, vm)
+        s, o = session.cmd_status_output(params.get("gagent_pkg_check_cmd"))
+        if s:
+            logging.info("qemu-ga is not installed.")
+            return 1
+        else:
+            logging.info("qemu-ga is already installed.")
+            return 0
```

Comment:
1. Suggest changing the function name to `_check_ga_pkg`, because it is a private function.
2. Return `True` or `False` instead of `1` or `0`. e.g. `return s == 0`.
3. The log can be moved into `self.setup()`, there is a better place to do that.
4. It is better to receive the `session` object from parameter instead of open a new one, otherwise should close the session once finished using.

Reply:
Thanks @luckyh 
I have updated as you suggested.